product_docs/docs/pgd/4/upgrades/index.mdx

```diff
@@ -58,9 +58,14 @@ all nodes are at the newer release. There should never be more than two versions
 of any component running at the same time, which means the new upgrade must not
 be initiated until the previous upgrade process has fully finished on all nodes.
 
-An upgrade process may take an extended period of time when the user decides
-caution is required to reduce business risk, though it's not recommended
-to run the mixed versions of the software indefinitely.
+An upgrade process can take more time when caution is required to reduce
```

Comment:
@djw-m would update this to match the version 5 language

Reply:
It does match the version 5 language *for the upgrades section*.